src/ProjectTask.php

```diff
@@ -1802,6 +1808,30 @@ public static function recalculatePercentDone($ID)
         return true;
     }
 
+    /**
+     * Recalculate the status of a project task based on the percent_done.
+     * @since 11.0.0
+     * @param array $input
+     */
+    public static function recalculateStatus(array $input)
```

Comment:
```suggestion
     * @since 11.0.0
     * @param array $input
     * @return int|false
     */
    public static function recalculateStatus(array $input): int|false
```

Let's add a return type hint